### DISCUSSION TITLE
Passwords Protected Share Links

### DISCUSSION THREAD

**Discussion by ryantcarter** [outline/outline#2610] [Ideas]
I'd like to make a suggestion of implementing passwords on shared links - shared links are awesome, and allow us to share internal documents externally with clients and stakeholders. 

That being said, it'd be nice to have an authentication layer on these so that we can ensure that our public links aren't visible to _anyone_. A simple password based auth would work nicely to add a level of protection to external links.

**Comment by kmanojkumar** [comment]
This is a very useful feature ensuring data is accessible to only those with whom the password is shared, and not anybody with the link.

**Comment by SangeloDev** [comment]
This would be a great addition!

**Reply by EduNetEurope** [reply]
@tommoor 
Is password-protected sharing on your roadmap?
